guides/source/active_record_validations.md

```diff
@@ -540,10 +551,6 @@ class Order < ApplicationRecord
 end
 ```
 
-If you validate the presence of an object associated via a `has_one` or
-`has_many` relationship, it will check that the object is neither `blank?` nor
-`marked_for_destruction?`.
-
```

Comment:
The new note can live just here after this line:

NOTE: If you want to ensure that the association it is both present and valid, you also need to use
`validates_associated`.

Reply:
@vipulnsward Much better! Thank you!